designer/client/src/components/graph/node-modal/fragment-input-definition/settings/variants/fields/ValidationFields.tsx

```diff
@@ -55,7 +55,10 @@ export default function ValidationFields({
                 <NodeInput
                     style={{ width: "70%" }}
                     value={validationFailedMessage}
-                    onChange={(event) => onChange(`${path}.valueCompileTimeValidation.validationFailedMessage`, event.currentTarget.value)}
+                    onChange={(event) => onChange(
+                        `${path}.valueCompileTimeValidation.validationFailedMessage`,
+                        event.currentTarget.value === "" ? null : event.currentTarget.value
```

Comment:
I don't think that it is a correct solution. `NodeInput` class name suggest that it is a kind of lightweight wrapper for DOM input's. We'd rather want to always have at least a blank string in the value. I think that the correct change should be moving this logic to some place where we prepare a request to backend. @JulianWielga am I right or wrong?

Reply:
Ahh, sorry now I'm get it. In node input you keep a blank string, you just change it during building of `validationFailedMessage`. It looks ok